rest/data/interface.go

```diff
@@ -77,8 +77,8 @@ type Connector interface {
 	// RestartBuild is a method to restart the build matching the same BuildId.
 	RestartBuild(string, string) error
 
-	// Find project variables matching given projectId.
-	FindProjectVarsById(string) (*restModel.APIProjectVars, error)
+	// Find project variables matching given projectId. If bool is set, returns only redacted values.
+	FindProjectVarsById(string, bool) (*restModel.APIProjectVars, error)
```

Comment:
I'm super dubious of adding boolan arguments to interface signatures? should we just have different methods? Is that obnoxious?

Reply:
Well we do have other functions with bools. If you _really_ want we could have two but `FindProjectVarsById` and `FindRedactedProjectVarsById` does seem like overkill.